dist/style/style.css

```diff
@@ -7887,20 +7888,23 @@ div.alpheios-color_schema_dark_class {
   div.alpheios-color_schema_dark_class .infl-cell--hl {
     background: #686868; }
   div.alpheios-color_schema_dark_class .alph_tooltip .tooltiptext {
-    background-color: #f2f2f2;
-    color: #282828; }
+    background-color: #fcfcfa;
+    color: #4E6476;
+    border: 1px solid #4E6476; }
   div.alpheios-color_schema_dark_class .alph_tooltip-top::after {
-    border-color: #f2f2f2 transparent transparent transparent; }
+    border-color: #4E6476 transparent transparent transparent; }
```

Comment:
Since we're using #4E6476 in multiple places maybe it's better to create a SCSS variable for it?

Reply:
Yes. it is a scss variable in _colorschema_dark.scss
.alph_tooltip-top::after {
		border-color: $alpheios-color_schema_dark_color transparent transparent transparent;
	}
dist/style/style.css - is compiled from these scss files